minesweeper_model/str_input.py

```diff
@@ -0,0 +1,28 @@
+field=[]
+mine_coord=[]
+x_len,y_len=0,0
+x_mine=0
+column=''
+
+while(True):
+	colum=str(input())
```

Comment:
You are receiving the input line-by-line from the user here. Can I please ask that you receive the input as a whole, that is, as a single string passed as a parameter into the function instead of reading it from the user line-by-line?

Reply:
oh! so do you mean the user will give input like -> "..x.\n..xx\nxx.x" 
which will be interpreted as -> 
..x.
..xx
xx.x
Did I interpret this correct?

Reply:
Yes, exactly

Reply:
user may use anything other than "\n" to provide input also, just let me know the input format.

Reply:
Only "\n" would be enough for now